app/controllers/registers_controller.rb

```diff
@@ -20,7 +20,7 @@ def index
   end
 
   def show
-    render json: @register
+    render json: @register, status: 200
```

Comment:
http://billpatrianakos.me/blog/2013/10/13/list-of-rails-status-code-symbols/

... status: :ok